build.gradle

```diff
@@ -1,54 +1,80 @@
 buildscript {
     repositories {
         jcenter()
+        maven { url "http://palantir.bintray.com/releases" }
     }
 
     dependencies {
-        classpath "com.palantir:gradle-baseline-java:${baselineVersion}"
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:${bintrayPlugin}"
-        classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0"
+        classpath 'com.netflix.nebula:nebula-dependency-recommender:3.6.3'
+        classpath 'gradle.plugin.com.palantir.configurationresolver:gradle-configuration-resolver-plugin:0.2.0'
+        classpath 'com.palantir.gradle.docker:gradle-docker:0.12.0'
+        classpath 'com.palantir.baseline:gradle-baseline-java:0.13.0-rc3'
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.14'
+        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
+        classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.0.0'
     }
 }
 
 plugins {
-    id 'com.palantir.docker-test-runner' version '0.1.1'
-    id 'com.palantir.git-version' version '0.4.0'
-    id 'org.inferred.processors' version '1.1.2'
+    id 'com.palantir.git-version' version '0.5.2'
+    id 'org.inferred.processors' version '1.2.3'
     id 'com.palantir.jacoco-full-report' version '0.3.0'
+    id 'org.unbroken-dome.test-sets' version '1.3.2'
 }
 
-apply plugin: 'com.palantir.baseline-config' // provides baselineUpdateConfig task
+repositories {
+    jcenter()
+    maven { url "http://palantir.bintray.com/releases" }
+}
+
+apply plugin: 'com.palantir.baseline-config'
 apply plugin: 'com.palantir.baseline-idea'
+apply plugin: 'org.inferred.processors'
 
-repositories {
-    jcenter() // needed for baselineUpdateConfig task
+dependencies {
+    baseline group: 'com.palantir.baseline', name: 'gradle-baseline-java-config', version: '0.13.0-rc3', ext: 'zip'
```

Comment:
prefer to leave buildscript dependency versions only in build.gradle, so you don't have to update buildscript deps across two files at least